raco/backends/backend_common.py

```diff
@@ -219,6 +231,10 @@ def visit_NaryFunction(self, expr):
                 type(expr).__name__,
                 *arglist))
 
+    def visit_CustomZeroaryOperator(self, expr):
```

Comment:
Nitpick: what's the distinction between "function" and "operator" in this context (since the binary form uses "function")?